packages/nx/src/utils/code-frames.ts

```diff
@@ -11,14 +11,17 @@ type NodeLocation = {
   start?: Location;
 };
 
+const markerString = (str: string) => pc.red(pc.bold(str));
+const messageString = (str: string) => pc.red(pc.bold(str));
+
 /**
  * Chalk styles for code frame token types.
  */
-function getDefs(chalk) {
```

Comment:
I wasn't sure why we passed `chalk` in since we already unconditionally import it here anyway, so I changed it to just reference `pc` directly

although we probably don't even need a `getDefs` and could just have a constant definition